package.json

```diff
@@ -21,43 +21,33 @@
     "Robin Berjon"
   ],
   "devDependencies": {
-    "@babel/core": "^7.9.0",
-    "@types/clipboard": "^2.0.1",
     "@types/pluralize": "0.0.29",
     "babel-eslint": "^10.1.0",
     "boxen": "^4.2.0",
     "chai": "^4.2.0",
     "chokidar": "^3.3.1",
     "clean-css": "^4.2.3",
-    "domReady": "github:requirejs/domReady#d85bdc38a6df868ead52ab3e80715aaf60e68765",
     "eslint": "^6.8.0",
     "eslint-config-prettier": "^6.10.1",
     "eslint-plugin-import": "^2.20.1",
     "eslint-plugin-jasmine": "^4.1.0",
     "eslint-plugin-prettier": "^3.1.2",
-    "handlebars": "^4.7.3",
```

Comment:
We might need to bring this back if the Geonovum folks finish migrating over their profile... they are still using it, I think.

Reply:
@thijsbrentjens Would you be needing handlebars?

Reply:
Oh, maybe not... all the templates are using hyperHTML! https://github.com/Geonovum/respec/tree/develop/src/geonovum/templates

Reply:
As an aside, looks like downstream is getting really out of date... @thijsbrentjens, we should look at moving more stuff over, as Geonovum might be missing out on important features and security updates.

Reply:
Thanks for confirming!